orchestration/dagster_orchestration/hca_orchestration/solids/validate_ingress.py

```diff
@@ -0,0 +1,46 @@
+from dagster import solid, String
+from dagster.core.execution.context.compute import AbstractComputeExecutionContext
+
+from hca_manage.common import ProblemCount
+
+from hca_manage.validation import run as validate
+
+@solid(
+    required_resource_keys={'data_repo_client'},
```

Comment:
🔧 This solid doesn't appear to need a data repo client, right? In which case I think we can remove the resource altogether from this pipeline.